### ISSUE TITLE
[BUG] - Append to Input does not work

### ISSUE THREAD

**Issue by DevinloperNL** [tonysamperi/ngx-mat-timepicker#5]
**Describe the bug**
Having a timepicker that appends to the input will still open a dialog in the middle of the screen with the buttons. The timepicker will not have a background either.

**To Reproduce**
```html
<mat-form-field appearance="outline">
  <mat-label>{{ item.label }}</mat-label>
  <input type="text" matInput [ngxMatTimepicker]="timepicker" format="24"
         [matTooltip]="item.toolTip" [required]="required" readonly [formControl]="formControlItem">

    <!-- Clear Timefield -->
    <button mat-icon-button matSuffix
            *ngIf="formControlItem.value && !formControlItem.disabled && !required" (click)="onClear($event)">
      <mat-icon>close</mat-icon>
    </button>

    <button mat-icon-button [disabled]="formControlItem.disabled" (click)="openFromIcon()" matSuffix>
      <mat-icon>
        watch_later_outlined
      </mat-icon>
    </button>
</mat-form-field>

<ngx-mat-timepicker [appendToInput]="true" #timepicker [append]></ngx-mat-timepicker>
````
```ts
import { NgxMatTimepickerComponent } from 'ngx-mat-timepicker/lib/components/ngx-mat-timepicker/ngx-mat-timepicker.component';

  @ViewChild('timepicker') public timepicker: NgxMatTimepickerComponent;

  /**
   * Lets the user click on the icon in the input.
   */
  public openFromIcon() {
    if (!this.formControlItem.disabled){
      this.timepicker.open();
    }
  }

  /**
   * Function to clear FormControl's value, called from the HTML template using the clear button
   *
   * @param $event - The Event's data object
   */
  public onClear($event: Event) {
    this.formControlItem.setValue(null);
  }
```

**Expected behavior**
The timepicker should append to the input like it does in the demo of the original package.
https://agranom.github.io/ngx-material-timepicker/ Refer to the Append to input section.

**Screenshots**
![image](https://user-images.githubusercontent.com/22426652/110807816-b2619b80-8283-11eb-81cc-baf498c7466e.png)

**Desktop (if execution):**
 - OS: MacOS Big Sur
 - Browser: Brave (Chromium)

** Versions
 - ngx-mat-timepicker version [10.0.0] (My project uses angular version 10)
 - @angular/core version [10.2.4]
 - @angular/material version [10.2.7]
 - typescript version [3.9.5]

**Additional context (optional)**
I would like to request two features aswell.
1. Export `NgxMatTimepickerComponent` and `NgxMatTimepickerRef` from the public api, so it removes the need for a deep import.
2. Make it possible to assign different colors from the Theme to different elements of the timepicker. For example, making the dial `warn` or making the numbers `accent`. Since the solution of the old package is not supported in yours.

**Comment by tonysamperi** [comment]
Can you make a blitz of the problem please?

**Comment by tonysamperi** [comment]
Anyway I give you a tip in the meantime

![image](https://user-images.githubusercontent.com/5957244/110923309-0c646e80-8321-11eb-9d94-54bc2cb6a184.png)

This stuff is useless. The button is disabled when the control is invalid, so there's no point in checking again inside the function.
So the best approach is to simply have `timepicker.open()` in your HTML, so you can also avoid using `ViewChild`.
Plus, if you really need to call the `open` in your callback, you can pass the timepicker from the HTML, so you don't need the `ViewChild` in any case.

**Comment by DevinloperNL** [comment]
The checking inside the function is from an old bit of code. And you're right, this could've been done way easier.

**Comment by DevinloperNL** [comment]
Any progress updates on this?

**Comment by tonysamperi** [comment]
@DevinloperNL yes, I'm working on the solution. As you may imagine, I work on this in my spare time, so I can't tell when the fix will be ready...
But let's say that we'll have at Easter tops!

**Comment by tonysamperi** [comment]
@DevinloperNL it's almost done! 
It took more time than expected, but I wanted to do it the right way....And this is way better than the original (faster and resilient).
I'll probably publish the update in a few hours!

**Comment by tonysamperi** [comment]
Fixed in v 11.0.2! 🎉
